lib/patch-series.ts

```diff
@@ -376,7 +376,8 @@ export class PatchSeries {
                 // Special-case GitGitGadget to send from
                 // "<author> via GitGitGadget"
                 replaceSender = "\""
-                    + onBehalfOf
+                    + onBehalfOf.replace(/^"(.*)"$/, "$1")
```

Comment:
Can `authorMatch[2]` ever be quoted (in the `isGitGitGadget` case)?  If not, should 
`PatchSeries.encodeSender(senderName)` at line 374 be `PatchSeries.encodeWords(senderName)`?  It would remove the need to dequote.

Reply:
> Can `authorMatch[2]` ever be quoted (in the `isGitGitGadget` case)?

Yes. The `authorMatch` variable contains matches from the `From:` lines generated by `git format-patch`. If `user.name` contains a `.` (as was in the case in the PR that was not sent to the mailing list), then that `From:` line indeed contains double quotes.

Reply:
> Yes. The `authorMatch` variable contains matches from the `From:` lines generated by `git format-patch`. If `user.name` contains a `.` (as was in the case in the PR that was not sent to the mailing list), then that `From:` line indeed contains double quotes.

Okay, so format-patch is also quoting.  The PR looks good.  Not sure of the chance of the second replace happening.  Would git allow double quotes in the user.name and if so, would it already be escaped?

This may explain why my initial PR for Git did not make it to the mailing list.  You had to forward it.  Looking at some of the emails I noticed an unquoted copy of my name so not sure if there is something else lurking but that would be a different problem.

Reply:
> Not sure of the chance of the second replace happening. Would git allow double quotes in the user.name and if so, would it already be escaped?

Neither am I, and hopefully we will never have to find out. I just wanted to err on the side of caution.

> This may explain why my initial PR for Git did not make it to the mailing list. You had to forward it. Looking at some of the emails I noticed an unquoted copy of my name so not sure if there is something else lurking but that would be a different problem.

Indeed! That could have been the problem all along...